Ports/freetype/package.sh

```diff
@@ -1,16 +1,16 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=freetype
-version=2.12.0
+version=2.12.1
 useconfigure=true
 use_fresh_config_sub=true
 config_sub_paths=("builds/unix/config.sub")
-files="https://download.savannah.gnu.org/releases/freetype/freetype-${version}.tar.gz freetype-${version}.tar.gz 7940a46eeb0255baaa87c553d72778c4f8daa2b8888c8e2a05766a2a8686740c"
+files="https://download.savannah.gnu.org/releases/freetype/freetype-${version}.tar.gz freetype-${version}.tar.gz efe71fd4b8246f1b0b1b9bfca13cfff1c9ad85930340c27df469733bbb620938"
 auth_type=sha256
 configopts=("--with-brotli=no" "--with-bzip2=no" "--with-zlib=no" "--with-harfbuzz=no" "--with-png=no")
 config_sub_paths=("builds/unix/config.sub")
 
 install() {
-    run make DESTDIR=${SERENITY_INSTALL_ROOT} "${installopts[@]}" install
-    ${CC} -shared -o ${SERENITY_INSTALL_ROOT}/usr/local/lib/libfreetype.so -Wl,-soname,libfreetype.so -Wl,--whole-archive ${SERENITY_INSTALL_ROOT}/usr/local/lib/libfreetype.a -Wl,--no-whole-archive
-    rm -f ${SERENITY_INSTALL_ROOT}/usr/local/lib/libfreetype.la
+    run make DESTDIR="${SERENITY_INSTALL_ROOT}" "${installopts[@]}" install
+    "${CC}" -shared -o "${SERENITY_INSTALL_ROOT}/usr/local/lib/libfreetype.so" -Wl,-soname,libfreetype.so -Wl,--whole-archive "${SERENITY_INSTALL_ROOT}/usr/local/lib/libfreetype.a" -Wl,--no-whole-archive
```

Comment:
```diff
-${CC}
+"${CC}"
```

This will break compiling with Clang, as we do `export CC="clang --target=${SERENITY_ARCH}-pc-serenity --sysroot=${SERENITY_BUILD_DIR}/Root"` and therefore rely on variable splitting. While it is a :hairyak: that we should eventually solve, it's probably out-of-scope for this PR.